### DISCUSSION TITLE
Dot separator instead of comma

### DISCUSSION THREAD

**Discussion by mateusz-wyka** [dm4t2/vue-currency-input#401] [Q&A]
Hi!

Is it possible to somehow change the comma separator to dot?

Current value: `5 999,00`
Desired value: `5 999.00`

**Comment by dm4t2** [comment]
The formatting is controlled by [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat) and depends on your configured `locale` and `currency`. What do you use?
